cliquet/permission/redis.py

```diff
@@ -79,6 +79,23 @@ def object_permission_principals(self, object_id, permission):
         return self._decode_set(members)
 
     @wrap_redis_error
+    def principals_accessible_objects(self, principals, permission,
```

Comment:
We should also pass the ``get_bound_permissions=None`` callback to get the permission inheritance.